src/server/database.js

```diff
@@ -0,0 +1,26 @@
+const MongoClient = require('mongodb').MongoClient;
+const url = 'mongodb://localhost:27017/epam';
+
+function checkUserInfo(email, password) {
+  MongoClient.connect(url, function(err, db) {
+    db.collection('users').findOne({email: email}, function(err, obj) {
+      if (obj !== null && obj.password === password) {
```

Comment:
we are storing encrypted password, this wont work. Check how is it encrypted in the signup flow